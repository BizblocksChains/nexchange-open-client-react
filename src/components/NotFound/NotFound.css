#not-found {
  padding: 150px 0;
  background: #f0f0f0;

  h1 {
    font-size: 50px;
    color: green;
  }

  h2 {
    font-size: 30px;
  }
}
