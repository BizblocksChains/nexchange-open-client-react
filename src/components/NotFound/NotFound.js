import React from 'react';
import { I18n } from 'react-i18next';
import './NotFound.css';

const NotFound = () => (
  <I18n ns="translations">
    {t => (
      <div id="not-found" className="text-center">
        <h1>404</h1>
        <h2>{t('error.notfound')}</h2>
      </div>
    )}
  </I18n>
);

export default NotFound;
