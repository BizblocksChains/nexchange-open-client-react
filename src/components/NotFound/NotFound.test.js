import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<NotFound />).dive();
    expect(wrapShallow).toMatchSnapshot();
  });
});
