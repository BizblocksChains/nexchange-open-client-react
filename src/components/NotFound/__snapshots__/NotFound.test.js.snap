// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <I18n
    ns="translations"
>
    [Function]
</I18n>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          404
</h1>,
        <h2>
          error.notfound
</h2>,
      ],
      "className": "text-center",
      "id": "not-found",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404",
        },
        "ref": null,
        "rendered": "404",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "error.notfound",
        },
        "ref": null,
        "rendered": "error.notfound",
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            404
</h1>,
          <h2>
            error.notfound
</h2>,
        ],
        "className": "text-center",
        "id": "not-found",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404",
          },
          "ref": null,
          "rendered": "404",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "error.notfound",
          },
          "ref": null,
          "rendered": "error.notfound",
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
